drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -279,6 +279,13 @@ config NRF700X_RPU_PS_IDLE_TIMEOUT_MS
 	int "RPU power save idle timeout in milliseconds"
 	default 10
 
+config NRF700X_RPU_EXTEND_TWT_SP
+	bool "Enable extending TWT service period"
+	help
+	  Enabling this feature extends TWT service period.
+	  Only accepted frames during SP will be send.
+	  New frames outside SP will not be send out.
+	  It uses EDCA to avoid dropping of frames, so consumes more power.
```

Comment:
```suggestion
     In case frames accepted before beginning of SP are not
     transmitted before the SP completes then typically they are
     dropped to conform to SP window as per specification i.e., no
     transmission outside SP window.
     
     This feature mitigates the frame loss by transmitting even after SP
     completion by using standard contention mechanism which is allowed
     in specification but not recommended. As the device is actively transmitting
     beyond SP, the power consumption increases depending on the amount
     of traffic available at the start of SP. 
     
     Please note that if a frame is sent after SP starts it will be queued and this 
     mechanism is not used.
```